content/rekor-v2-beta.md

```diff
@@ -0,0 +1,53 @@
++++
+title = "Rekor v2 Beta - Cheaper to run, simpler to maintain"
+date = "2025-09-12"
+tags = ["sigstore","rekor","transparency"]
+draft = false
+author = "Hayden Blauzvern, Google"
+type = "post"
++++
+
+We are excited to announce the beta release of Rekor v2!
+
+[Rekor v2](https://github.com/sigstore/rekor-tiles) is a redesigned and modernized
+[Rekor](https://github.com/sigstore/rekor), Sigstore's signature transparency log,
+transitioning its backend to a modern, tile-backed transparency log implementation to simplify
+maintenance and lower operational costs. Learn more about Rekor v2 in our
+[previous blog post announcing alpha](https://blog.sigstore.dev/rekor-v2-alpha/).
+
+With the beta release, we have added support for Rekor v2 upload and verification to the
+[Go](https://github.com/sigstore/sigstore-go), [Python](https://github.com/sigstore/sigstore-python),
+and [Java](https://github.com/sigstore/sigstore-java) clients, and released a new version of
+[Cosign](https://github.com/sigstore/cosign/releases/tag/v2.6.0) that supports Rekor v2 upload and verification.
+We have also released a new version of the
+[conformance test suite](https://github.com/sigstore/sigstore-conformance/releases/tag/v0.0.20) for client
```

Comment:
could link to the conformance project -- the latest release notes only contain a subset of all rekor v2 tests

```suggestion
[conformance test suite](https://github.com/sigstore/sigstore-conformance) for client
```